### PR TITLE
Add restartPlugin function to EdgeProvider and PluginView to regenera…

### PR DESCRIPTION
**Pull request by sfncook**
…te webView component on demand

#### PR Requirements

If you have made **any** visual changes to the GUI. Make sure you have:
- [x] Tested on iOS Tablet
- [x] Tested on small Android
- [ ] n/a

### CODE REVIEW

File: src/components/scenes/PluginViewScene.js [hunk 1]
```diff
@@ -146,6 +152,8 @@ class PluginView extends React.Component<Props> {
       this._webview = element
       setRef(element)
     }
+
+    this.state = { webViewKey: new Date().getTime() }
```
Comment (swansontec) [inline src/components/scenes/PluginViewScene.js]: Start at 0 and count up?

File: src/components/scenes/PluginViewScene.js [hunk 2]
```diff
@@ -111,11 +111,15 @@ type Props = {
   state: State
 }
 
+type PluginViewState = {
```
[the remark this answers is not in the document]
Comment (sfncook) [reply src/components/scenes/PluginViewScene.js]: Done.

File: src/components/scenes/PluginViewScene.js [hunk 3]
```diff
@@ -128,8 +132,10 @@ class PluginView extends React.Component<Props> {
     // Set up the plugin:
     const { dispatch, plugin, state } = this.props
 
+    this.restartPlugin = this.restartPlugin.bind(this)
```
[the remark this answers is not in the document]
Comment (sfncook) [reply src/components/scenes/PluginViewScene.js]: Ah yes.  That's one of those subtleties I often forget about.  Thanks.  Done.

File: src/components/scenes/PluginViewScene.js (hunk 1 above)
Comment (sfncook) [reply src/components/scenes/PluginViewScene.js]: Oh ok.  Done.

**Comment by swansontec** [comment]
Resolved some conflicts, and now this is seems good to merge!

**Pull request merged**